server/middleware.json

```diff
@@ -47,6 +47,32 @@
       }
     }
   },
+  "auth:after": {
+    "./middleware/routes-acl": {
+      "params": {
+        "rules": [
+          {
+            "route": "/api/notes",
+            "acl": {
+              "accessType": "*",
+              "principalType": "ROLE",
+              "principalId": "$everyone",
+              "permission": "DENY"
+            }
+          },
+          {
+            "route": "/api/notes/bounce",
+            "acl": {
+              "accessType": "*",
+              "principalType": "ROLE",
+              "principalId": "$everyone",
+              "permission": "ALLOW"
+            }
+          }
+        ]
+      }
+    }
+  },
```

Comment:
I also want to have the ability to create `scopes` to group routes and the ACL should apply to `scopes`